addons/monit/monit_checks_configurations/00_packetfence.tt

```diff
@@ -114,11 +114,16 @@ check process packetfence-redis_queue with pidfile /usr/local/pf/var/run/redis_q
     if failed host 127.0.0.1 port 6380 protocol redis for 5 cycles then alert
     if 3 restarts within 10 cycles then alert
 
+check process packetfence-winbind with pidfile /usr/local/pf/var/run/winbindd-wrapper.pid
```

Comment:
Wrap this into a condition (if there are domain(s) configured). Otherwise, the wrapper should not be managed / started.
Something like 
[% IF DOMAINS.size > 0 %]
Should work